### PR TITLE
Translation-ja: Add trusted-committer/ja/02-ensuring-product-quality

### PR DESCRIPTION
**Pull request by ystk**
This is the second PR for #367 .

### CODE REVIEW

File: trusted-committer/ja/02-ensuring-product-quality-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+これが先程「Trusted Committersはコードを本番環境に近づけることができる」と言った意味になります。
+Trusted Committersは、PR中にコントリビューションがゴールラインを越えられるように、 https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] を支援する必要もあります。
+
+そうは言っても、最終的にそれを実現するのはコントリビューターの仕事です。
+Trusted Committerの仕事は、デフォルトですべてのコントリビューションを受け入れるのではなく、品質と範囲に関して定義された基準を満たすものだけを受け入れることです。
+また、Trusted Committerは、PRで https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] のサポートに多くの時間を費やすことになっても、コントリビューターのコードをできるだけ「適合する」ように書き直すことは避けるべきです。
+Trusted Committerは、長期的な視点を持ち、このような支援がコミュニティの寿命への投資で、長期的にはコミュニティの開発速度を向上させることを理解しています。
+
+時々、プロジェクトの要件や制限が事前にわからず、開発中に発見されることがあります。
+Trusted Committerは、これらの発見が捕捉されドキュメント化されたことを https://innersourcecommons.org/resources/learningpath/product-owner/index[_プロダクトオーナー_] と https://innersourcecommons.org/resources/learningpath/contributor/index[_コントリビューター_] の両方に確認する責任もあります。
```
Comment (johtani) [inline trusted-committer/ja/02-ensuring-product-quality-ja.asciidoc:28]: 「これらの発見をプロダクトオーナーとコントリビューターのために捕捉し、ドキュメントにすることを確認する責任もあります。」のほうがしっくりくるかなと。どうでしょう？

**Review by johtani — CHANGES_REQUESTED** [review]
@ystk, I left a comment.
Please review it.

**Comment by rrrutledge** [comment]
@johtani thank you so much for this review!  You've been so reliable and helpful for reviewing this text.  We are very grateful.  I would really like to make sure that you are recognized for your work and service on our website.  We have a contributors list that is auto-generated off of the list of committers to the file.  I believe that if you leave a piece of feedback as a "commit suggestion" that is then accepted by Yoshi then your name will show up in the commit history and on the website.  If you ever have any feedback that could be given as a commit suggestion then place do so that you can show up with the credit that you've earned through your service.

**Comment by johtani** [comment]
@rrrutledge thank you for your comment. I haven't known "commit suggestion"! I'll do that next time :)

File: trusted-committer/ja/02-ensuring-product-quality-ja.asciidoc (hunk 1 above)
Comment (ystk) [reply trusted-committer/ja/02-ensuring-product-quality-ja.asciidoc:28]: ありがとうございます。

**Comment by ystk** [comment]
Thanks for the review, @johtani . Please check it, again.

**Review by johtani — APPROVED** [review]
LGTM!

**Pull request merged**